GDJS/Runtime/websocket-debugger-client/websocket-debugger-client.js

```diff
@@ -1,14 +1,64 @@
 /**
- * This DebuggerClient connects to a websocket server and can dump
+ * An client side implementation of the Debugger
+ * @interface
+ */
+gdjs.DebuggerClient = function() {} // Necessary for Interface definition 
+
+/**
+ * Update a value of the {@link RuntimeGame} instance.
```

Comment:
`Update a value, specified by a path starting from the {@link RuntimeGame} instance.`